packages/esm-reports-app/src/components/overview.component.tsx

```diff
@@ -24,17 +26,15 @@ import {
   userHasAccess,
   useSession,
 } from '@openmrs/esm-framework';
-import { Calendar, Download, Play, Save, TrashCan } from '@carbon/react/icons';
-import styles from './reports.scss';
-import { DEFAULT_PAGE_NUMBER, DEFAULT_PAGE_SIZE, DEFAULT_PAGE_SIZES } from './pagination-constants';
 import { closeOverlay, launchOverlay } from '../hooks/useOverlay';
-import RunReportForm from './run-report/run-report-form.component';
-import Overlay from './overlay.component';
-import ReportStatus from './report-status.component';
 import { COMPLETED, RAN_REPORT_STATUSES, SAVED } from './report-statuses-constants';
-import ReportOverviewButton from './report-overview-button.component';
+import { DEFAULT_PAGE_NUMBER, DEFAULT_PAGE_SIZE, DEFAULT_PAGE_SIZES } from './pagination-constants';
 import { PRIVILEGE_SYSTEM_DEVELOPER } from '../constants';
-import classNames from 'classnames';
+import Overlay from './overlay.component';
+import ReportOverviewButton from './report-overview-button.component';
+import ReportStatus from './report-status.component';
+import RunReportForm from './run-report/run-report-form.component';
```

Comment:
Shouldn't components be before constants? Since components would usually come before types.